src/components/remote-promo-sheet/RemotePromoSheet.tsx

```diff
@@ -96,9 +97,9 @@ export function RemotePromoSheet() {
     }
   };
 
-  const externalNavigation = useCallback(() => {
-    Linking.openURL(data?.promoSheet?.primaryButtonProps.props.url);
-  }, [data?.promoSheet?.primaryButtonProps.props.url]);
+  const externalNavigation = useCallback(async () => {
+    await openInBrowser(data?.promoSheet?.primaryButtonProps.props.url);
+  }, [data?.promoSheet?.primaryButtonProps.props.url, openInBrowser]);
```

Comment:
note to self to follow-up on this to serve a way for remote promo sheets to open either in-app or outside for external links